appveyor.yml

```diff
@@ -15,26 +15,26 @@ environment:
     #   VS_VERSION: "9 2008"
     #   ARCH: Win64
     # Python 3.4
-    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2015
+    - APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2013
```

Comment:
Why use this image @bemcdonnell ?

Reply:
@goanpeca, I suppose we don't have to.. I was experiementing with trying to get OpenMP to work on VS2010

Reply:
Well.. it does not matter much, just wondering why.

Reply:
Do you want me to revert that change?

Reply:
Naahh